events/tests.py

```diff
@@ -127,3 +126,52 @@ def test_exist_event_participant():
 def test_invalid_register_user_twice(persist_event_participant):
     with pytest.raises(Exception, match='user already exist in meeting'):
         persist_event_participant.save()
+
+
+@pytest.fixture
+def possible_meeting0(event_participant_not_creator):
+    return PossibleMeeting(participant_id=event_participant_not_creator,
+                           date_time_start=DATE_TIME_START, date_time_end=DATE_TIME_END)
+
+
+@pytest.fixture
+def persist_possible_meeting(possible_meeting0):
+    possible_meeting0.participant_id.user_id.save()
+    possible_meeting0.participant_id.event_id.save()
+    possible_meeting0.participant_id.save()
+    possible_meeting0.save()
+    return possible_meeting0
+
+
+@pytest.mark.django_db
+def test_persist_possible_meeting(persist_possible_meeting):
+    assert persist_possible_meeting in PossibleMeeting.objects.all()
+
+
+@pytest.mark.django_db
+def test_delete_persist_possible_meeting(persist_possible_meeting):
+    persist_possible_meeting.delete()
+    assert persist_possible_meeting not in PossibleMeeting.objects.all()
+
+
+@pytest.mark.django_db
+def test_delete_participant_deletes_possible_meeting(persist_possible_meeting):
+    persist_possible_meeting.participant_id.delete()
+    assert persist_possible_meeting not in PossibleMeeting.objects.all()
+
+
+@pytest.mark.django_db
+def test_persist_possible_meeting_Validation_Error(persist_possible_meeting):
+    with pytest.raises(ValidationError):
+        persist_possible_meeting.date_time_start = DATE_TIME_END
+        persist_possible_meeting.date_time_end = DATE_TIME_START
+        persist_possible_meeting.save()
+
```

Comment:
please add test of unique possible meeting.
as you can see, I added the same 2 POSSIBLE MEETING:
testUser1@mta.ac.il - event1 - 2022-03-24 11:11:11+00:00 - 2022-04-01 14:18:40+00:00
testUser1@mta.ac.il - event1 - 2022-03-24 11:11:11+00:00 - 2022-04-01 14:18:40+00:00

Reply:
@kobihk 
HI did you mean change the printing format of the variables "date time"
or change how the variable is saved because the later cant be changed...

Reply:
Done